clang/test/Driver/fp-model.c

```diff
@@ -64,7 +64,8 @@
 
 // RUN: %clang -### -ffp-model=strict -fdenormal-fp-math=preserve-sign,preserve-sign -c %s 2>&1 \
```

Comment:
You can use `-### -Werror` to ensure that there is no warning.

(In the absence of a `--target=`, you probably want to check a few different targets that the warning is printed regardless of the default target triple.)

Reply:
I didn't intend to change the logic of this test. The behavior here should be target-independent now.